src/controller/discoverymgr/discovery_test.go

```diff
@@ -753,3 +753,7 @@ func TestGetDeviceID(t *testing.T) {
 		}
 	})
 }
+
+func getDeviceID() (string, error) {
```

Comment:
I found that getDeviceID() changed to GetDeviceID() from PR #158. 
I think we should use GetDeviceID() rather than implementing getDeviceID() for the testing only.
What do you think?

Reply:
@ayush-kr @t25kim Seems valid. Now we should call `GetDeviceID()` instead of `getDeviceID()`.

@ayush-kr It seems to me that this PR is about to refactor the current codes to migrate the `MNEDC` server/client interrupt logic to the existing `sigmgr`, and this `GetDeviceID()` is a separate matter. What do you think? If yes, we would like to suggest you to separate this change to a new PR, so that every PR is per a single change / feature basis (as described in our official contribution guideline from [CONTRIBUTING.md](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/.github/CONTRIBUTING.md).

Reply:
Ok, I will remove the changes for GetDeviceID() from this PR.